scripts/pylint_extensions.py

```diff
@@ -90,12 +88,9 @@ def visit_call(self, node):
         parameters = []
         parameter_name_to_index = {}
         for i, arg in enumerate(called.args.args):
-            if isinstance(arg, astroid.Tuple):
```

Comment:
Each argument in the func definition should be a variable not a tuple. I can't think how this line be called.

Reply:
Can you elaborate more about this?

Reply:
Oops I was able to test it. Don't know what happened that time. Sorry!